code/app/src/main/java/com/example/goldencarrot/views/EntrantEditUserDetailsView.java

```diff
@@ -77,63 +120,7 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean b) {
             }
         });
 
-        saveButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
This is not gone, just split up and made into methods to help streamline things a bit better imo